android/src/main/java/com/onegini/mobile/sdk/reactnative/managers/CustomRegistrationActionManager.kt

```diff
@@ -0,0 +1,37 @@
+package com.onegini.mobile.sdk.reactnative.managers
+
+import com.onegini.mobile.sdk.reactnative.handlers.customregistration.CustomRegistrationAction
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class CustomRegistrationActionManager @Inject constructor() {
+
+    private val customRegistrationActions = ArrayList<CustomRegistrationAction>()
+
+    fun getCustomRegistrationAction(id: String?): CustomRegistrationAction? {
+        for (action in customRegistrationActions) {
+            if (action.id == id) {
+                return action
+            }
+        }
+        return null
```

Comment:
you could simplify it to something like this:
`return customRegistrationActions.firstOrNull { it.id == id }`